internal/auth0/user/resource.go

```diff
@@ -153,6 +153,9 @@ func NewResource() *schema.Resource {
 				Optional:    true,
 				Elem:        &schema.Schema{Type: schema.TypeString},
 				Description: "Set of IDs of roles assigned to the user.",
+				Deprecated: "This attribute is deprecated and it will be removed in a future version. " +
+					"Migrate to the `auth0_user_roles` resource to manage user roles instead. " +
```

Comment:
It's nice to keep this deprecation warning concise, but we could also advertise the eventual `auth0_user_role` too. WDYT?